raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -1617,7 +1660,7 @@ def test_do_not_claim_an_almost_expiring_lock_if_a_payment_didnt_occur():
         assert not any(
             event
             for event in new_iteration.events
-            if not isinstance(event, EventUnlockFailed)
+            if not isinstance(event, (EventUnlockFailed, ContractSendSecretReveal))
```

Comment:
The semantics changed, the test was for a previous case were the channel was closed because the node was greedy, with the SecretRegistry the channel doesnt need to be closed anymore to unlock on-chain. So I propose keeping the test, but renaming it from `test_do_not_claim_an_almost_expiring_lock_if_a_payment_didnt_occur` to `test_do_not_close_channel_because_of_greediness`, adding a short description for why the test exists, and rewrite this line to:

```python
assert not must_contain_entry(new_iteration.events, ContractSendChannelClose, {})
```